test/e2e/tests/custom-token-add-approve.spec.js

```diff
@@ -1,362 +1,506 @@
 /* eslint-disable mocha/no-skipped-tests */
```

Comment:
@mirjanaKukic can we remove this line now? I don't believe its needed anymore

Reply:
Done.